### PR TITLE
Move git2 binary to a subfolder

### PR DESCRIPTION
**Pull request by sharwell**
Small cleanup for external binaries that get included in the package.

### CODE REVIEW

File: GitDiffMargin/GitDiffMargin.csproj [hunk 1]
```diff
@@ -66,9 +67,7 @@
       GitDiffMargin.Commands references this project using a ProjectReference.
     -->
     <PackageReference Include="LibGit2Sharp" Version="0.26.2" PrivateAssets="all" />
-    <PackageReference Include="LibGit2Sharp.NativeBinaries" Version="2.0.306" ExcludeAssets="build" PrivateAssets="all">
-      <IncludeAssets>runtime; compile; native; contentfiles; analyzers</IncludeAssets>
-    </PackageReference>
+    <PackageReference Include="LibGit2Sharp.NativeBinaries" GeneratePathProperty="true" Version="2.0.306" ExcludeAssets="build" PrivateAssets="all" />
```
Comment (sharwell) [inline GitDiffMargin/GitDiffMargin.csproj:70]: 📝 Updating NuGet to 5.3.1 means `GeneratePathProperty="true"` is recognized and produces a new build property `PkgLibGit2Sharp_NativeBinaries` which I used for referencing the package location in the cache.

Comment (laurentkempe) [reply GitDiffMargin/GitDiffMargin.csproj:70]: Nice [learning](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#generatepathproperty)

File: GitDiffMargin/GitDiffMargin.csproj [hunk 2]
```diff
@@ -118,8 +117,8 @@
     <Compile Include="Core\WeakReference.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="$(NuGetPackageRoot)\libgit2sharp.nativebinaries\2.0.306\runtimes\win-x86\native\git2-106a5f2.dll">
-      <Link>x86\git2-106a5f2.dll</Link>
+    <Content Include="$(PkgLibGit2Sharp_NativeBinaries)\runtimes\win-x86\native\$(libgit2_filename).dll">
+      <Link>x86\$(libgit2_filename).dll</Link>
```
Comment (laurentkempe) [inline GitDiffMargin/GitDiffMargin.csproj:121]: Where is the `libgit2_filename` defined?

Comment (laurentkempe) [reply GitDiffMargin/GitDiffMargin.csproj:121]: Found it 
image

File: GitDiffMargin.Extension/source.extension.vsixmanifest [hunk 3]
```diff
@@ -3,7 +3,7 @@
     <Metadata>
         <Identity Id="GitDiffMargin..5ef9f816-d9ce-45cd-a8fa-cd67a807df0e" Version="3.9.0.0" Language="en-US" Publisher="Laurent Kempé" />
         <DisplayName>Git Diff Margin</DisplayName>
-        <Description xml:space="preserve">Git Diff Margin displays live Git changes of the currently edited file on Visual Studio margin and scroll bar. Supports Visual Studio 2012 through Visual Studio 2019 Preview.</Description>
+        <Description xml:space="preserve">Git Diff Margin displays live Git changes of the currently edited file on Visual Studio margin and scroll bar. Supports Visual Studio 2012 through Visual Studio 2019.</Description>
```
Comment (laurentkempe) [inline GitDiffMargin.Extension/source.extension.vsixmanifest:6]: 👍

File: GitDiffMargin.Extension/source.extension.vsixmanifest [hunk 4]
```diff
@@ -13,16 +13,16 @@
         <!--<Tags></Tags>-->
     </Metadata>
     <Installation>
-        <InstallationTarget Version="[11.0,17.0)" Id="Microsoft.VisualStudio.Community" />
-        <InstallationTarget Version="[11.0,17.0)" Id="Microsoft.VisualStudio.Pro" />
+        <InstallationTarget Version="[11.0,18.0)" Id="Microsoft.VisualStudio.Community" />
```
Comment (laurentkempe) [inline GitDiffMargin.Extension/source.extension.vsixmanifest:16]: What is version 18.0?

**Review by laurentkempe — APPROVED** [review]
Thanks for the PR @sharwell I just have one question about the version!

File: GitDiffMargin.Extension/source.extension.vsixmanifest (hunk 4 above)
Comment (sharwell) [reply GitDiffMargin.Extension/source.extension.vsixmanifest:16]: I'll come back to answer this one as soon as I can

Comment (laurentkempe) [reply GitDiffMargin.Extension/source.extension.vsixmanifest:16]: I thought it was the case 😉

**Pull request merged**